src/main/java/de/creelone/dismine/TeleportManager.java

```diff
@@ -0,0 +1,84 @@
+package de.creelone.dismine;
+
+import java.io.File;
+import java.util.UUID;
+
+import org.bukkit.Location;
+import org.bukkit.configuration.file.YamlConfiguration;
+
+import de.creelone.dismine.util.TeleportLocation;
+
+public class TeleportManager {
+	
+	private static File warpsFile = new File("plugins/Dismine/warps.yml");
+	private static YamlConfiguration warps = YamlConfiguration.loadConfiguration(warpsFile);
+
+	private static File homesFile = new File("plugins/Dismine/homes.yml");
+	private static YamlConfiguration homes = YamlConfiguration.loadConfiguration(homesFile);
+
+	public static void addWarp(TeleportLocation loc) {
+		warps.set(loc.getName(), loc.getLocation());
+		saveWarps();
+	}
+
+	public static void delWarp(String name) {
+		warps.set(name, null);
+		saveWarps();
+	}
+
+	public static Location getWarp(String name) {
+		return warps.getLocation(name);
```

Comment:
you sure there is a getLocation() function?
looks like a copy paste error, but didn't check the docs, so idk
and if there actually is a getLocation() function, shouldn't you try-catch it?

Reply:
If you call getWarp(...) with an invalid warp name it will return null, its fine and handled by the / command